protocol/x/affiliates/keeper/keeper.go

```diff
@@ -224,3 +226,7 @@ func (k Keeper) UpdateAffiliateTiers(ctx sdk.Context, affiliateTiers types.Affil
 	// staking requirements hold in UpdateAffiliateTiers
 	store.Set([]byte(types.AffiliateTiersKey), k.cdc.MustMarshal(&affiliateTiers))
 }
+
+func (k Keeper) SetRevShareKeeper(revShareKeeper types.RevShareKeeper) {
+	k.revShareKeeper = revShareKeeper // nolint
```

Comment:
`k` is a copy of the input keeper here so this assignment doesn't take any affect. This is probably why you are getting a lint error

Reply:
Good catch - i was getting `unused write to field revShareKeeper` which disappeared if I added another statement which uses revsharekeeper in the file.  so assumed it wasnt linting between files correctly.